grails-app/controllers/org/pih/warehouse/user/UserController.groovy

```diff
@@ -118,6 +118,10 @@ class UserController {
         log.info "attempt to save the user; show form with validation errors on failure"
         def userInstance = new User(params)
 
+        if (!params.hasProperty("active")) {
```

Comment:
Is this part required if there is a default value on it?

Reply:
the default value for _active_ on **Person**  is set to `True` which is inherited by **User** but when creating a new user default value should be `False`

Reply:
Could you add this as a comment above?